src/Processing.js

```diff
@@ -8049,6 +8051,13 @@
           j = i % PConstants.SINCOS_LENGTH;
           p.vertex(centerX + cosLUT[j] * hr, centerY + sinLUT[j] * vr);
         }
+
+        if (mode === PConstants.CHORD)
```

Comment:
we can extend this to `(mode === PConstants.CHORD || mode === PConstants.OPEN)`, as the vertex information is the same for both (the only difference between them is whether to stroke the last connection or not)